deploy/cic-spec.yaml

```diff
@@ -580,6 +580,93 @@ paths:
         passthroughBehavior: "when_no_match"
         contentHandling: "CONVERT_TO_TEXT"
         type: "aws"
+  
+  /session-config:
+    get:
+      operationId: getSessionConfig
+      summary: Return journey type for a session
+      tags:
+        - Backend - Common CRI Lambda
+      parameters:
+        - $ref: "#/components/parameters/SessionHeader"
```

Comment:
I see you've changed everything to `x-govuk-signin-session-id`, I think this also needs to change to `SessionId`

Reply:
Good catch. Thanks